doc/previous-work.md

```diff
@@ -0,0 +1,11 @@
+# Previous Work / References
```

Comment:
Would you be ok deleting this? I think we have these linked in the migration issue, and these repos will disappear in time.

Reply:
I'm OK with what you think is best.  I originally included just because I wanted to credit/cite the work upon which this is being built. If that's all going away with this as the replacement, then this would seem to be unnecessary (as it will point to archived/deleted material).